docs/source/advanced/using-graphbolt.rst

```diff
@@ -0,0 +1,169 @@
+
+.. code-block:: bash
+
+    ls /tmp/acm_graphbolt/part0
+    edge_feat.dgl  fused_csc_sampling_graph.pt  graph.dgl  node_feat.dgl
+
+The file ``fused_csc_sampling_graph.pt`` is the new representation that GraphBolt will use during training
+to optimized data loading.
+
+GSPartition with GraphBolt
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to GConstuct, when running distributed graph partitioning with the GSPartition module you can
+provide the ``--use-graphbolt true`` argument to convert the resulting partitions to GraphBolt
+at the end of partitioning. You'll need your data to be in DGL's "chunked graph format",
+which can be produced by feeding your raw data and GConstuct configuration to
+:ref:`GSProcessing <gsprocessing_prerequisites_index>`.
+
+.. code-block:: bash
+
+    python -m graphstorm.gpartition.dist_partition_graph \
+        --input-path "${INPUT_PATH}" \
+        --ip-config ip_list.txt \
+        --metadata-filename chunked_graph_meta.json \
+        --num-parts 2 \
+        --output-path "$DIST_GRAPHBOLT_PATH" \
+        --ssh-port 2222 \
+        --use-graphbolt "true"
+
+
+Running training and inference tasks with GraphBolt enabled
+-----------------------------------------------------------
+
+Now that we have prepared our data we can run a training job, with GraphBolt enabled.
+We run the same node classification task as in the original guide by adding the
+`--use-graphbolt true` argument to enable GraphBolt:
+
+.. code-block:: bash
+
+    python -m graphstorm.run.gs_node_classification \
+              --part-config /tmp/acm_graphbolt/acm.json \
+              --num-epochs 30 \
+              --num-trainers 1 \
+              --num-servers 1 \
+              --cf ./examples/use_your_own_data/acm_nc.yaml \
+              --save-model-path /tmp/acm_nc_graphbolt/models \
+              --node-feat-name paper:feat author:feat subject:feat \
+              --use-graphbolt true
+
+Similarly, we can run an inference task with GraphBolt enabled as such:
+
+.. code-block:: bash
+
+    python -m graphstorm.run.gs_node_classification \
+              --inference \
+              --part-config /tmp/acm_graphbolt/acm.json \
+              --num-trainers 1 \
+              --num-servers 1 \
+              --cf ./examples/use_your_own_data/acm_nc.yaml \
+              --node-feat-name paper:feat author:feat subject:feat \
+              --restore-model-path /tmp/acm_nc_baseline/models/epoch-29 \
+              --save-prediction-path  /tmp/acm_nc_graphbolt/predictions \
+              --use-graphbolt true
+
+Converting existing partitioned graphs to the GraphBolt format
+--------------------------------------------------------------
+
+If you have a partitioned graph that you would like to
+run training or inference on, using GraphBolt, you can directly convert that
+data using our GraphBolt conversion entry point:
+
+.. code-block:: bash
+
+    # For testing, let's remove the converted graph file
```

Comment:
Do we really need the line of 161 and 162? And,

if the `fused_csc_sampling_graph.pt` is already there, will its existence cause an error?